userguide/payment/includes/payment-states.adoc

```diff
@@ -55,29 +55,30 @@ Note that the first 3 cases are normal cases but the last 2 are errors that are
 
 If a payment is aborted by a control plugin, a payment attempt row is recorded with a state of ABORTED (no payment nor payment transaction is created). Kill Bill will return a 422 HTTP code.
 
+[[Janitor]]
 === Janitor
 
-PENDING and UNKNOWN transaction states should be temporary: the payment plugin needs to eventually tell Kill Bill what the final state of these transactions should be. This can be done explicitly by the plugin by calling the `notifyPendingTransactionOfStateChanged` PaymentApi (for instance, when receiving a webhook event from the gateway) or implicitly by the Janitor.
+PENDING and UNKNOWN transaction states should be temporary: the payment plugin needs to eventually tell Kill Bill what the final state of these transactions should be. This can be done explicitly by the plugin by calling the https://github.com/killbill/killbill-api/blob/4ae1c343a593de937415e21feecb9f5405037fa3/src/main/java/org/killbill/billing/payment/api/PaymentApi.java#L402[`PaymentApi.notifyPendingTransactionOfStateChanged`] (for instance, when receiving a webhook event from the gateway) or implicitly by the Janitor.
 
-The Janitor refers to a set of internal Kill Bill features to help ensure transactions end up in a terminal state: these are the Janitor notification queue and the on-the-fly Janitor. They both work by calling the plugin (via `getPaymentInfo` PaymentPluginApi) and updating the state owned by Kill Bill if it has changed. It is imperative payment plugin developers take great care when implementing this API for this mechanism to work (see our Payment Plugin developer documentation for more details).
+The Janitor refers to a set of internal Kill Bill features to help ensure transactions end up in a terminal state: these are the Janitor notification queue and the on-the-fly Janitor. They both work by calling the plugin (via https://github.com/killbill/killbill-plugin-api/blob/d9eca5af0e37541069b1c608f95e100dbe13b301/payment/src/main/java/org/killbill/billing/payment/plugin/api/PaymentPluginApi.java#L144[`PaymentPluginApi.getPaymentInfo`]) and updating the state owned by Kill Bill if it has changed. It is imperative payment plugin developers take great care when implementing this API for this mechanism to work (see our Payment Plugin developer documentation for more details).
```

Comment:
```suggestion
The Janitor refers to a set of internal Kill Bill features to help ensure transactions end up in a terminal state; these are the Janitor notification queue and the on-the-fly Janitor. They both work by calling the plugin (via https://github.com/killbill/killbill-plugin-api/blob/d9eca5af0e37541069b1c608f95e100dbe13b301/payment/src/main/java/org/killbill/billing/payment/plugin/api/PaymentPluginApi.java#L144[`PaymentPluginApi.getPaymentInfo`]) and updating the state owned by Kill Bill if it has changed. It is imperative payment plugin developers take great care when implementing this API for this mechanism to work (see our Payment Plugin developer documentation for more details).
```
The semi-colon is correct to use to join two closely related *complete* sentences (each can stand alone). The colon is appropriate to use in front of a short list or, in the above case, "The Janitor refers to a set of internal Kill Bill features to help ensure transactions end up in a terminal state: the Janitor notification queue and the on-the-fly Janitor." (I removed "these are.")